src/main/java/bio/terra/cloudfiletodatastore/deltalayer/DeltaLayerBigQueryWriter.java

```diff
@@ -0,0 +1,78 @@
+package bio.terra.cloudfiletodatastore.deltalayer;
```

Comment:
I'm using the SQL-like BQ API in this class and in DeltaLayerBQInsertGenerator since it features prominently in Google's doc and is fairly intuitive, but there are two alternatives which may offer advantages.